pySDC/tests/test_projects/test_DAE/test_misc.py

```diff
@@ -8,7 +8,7 @@ def test_problem_class_main():
     from pySDC.projects.DAE.problems.simple_DAE import simple_dae_1
 
     # initialize problem parameters
-    problem_params = dict()
+    problem_params = {'nvars': 3}
```

Comment:
Why do you need to pass `nvars` here? Didn't you remove it everywhere? Does this class support different number of variables?

Reply:
Maybe this whole test does not make any sense, so I'll remove that. Complaints?

Reply:
Fine by me. I think we don't need to overdo the tests. Testing math is more important than testing API.